hw_19.09.23/task3.py

```diff
@@ -0,0 +1,66 @@
+class Product:
+    def __init__(self, type_product, name, price):
+        self.type_product = type_product
+        self.name = name
+        self.price = price
+
+    def __repr__(self):
+        return f"{self.type_product}:{self.name}:{self.price}"
+
+
+class ProductStore:
+
+    def __init__(self):
+        self.storage = {}
+        self.income = 0
+
+    def add(self, product, amount):
+        product.price *= 1.3
```

Comment:
"premium" probably means the same as "discount", but OK, I'm not 100% sure what they meant by that in the task description